docs/develop/edit_a_module.md

```diff
@@ -45,15 +47,15 @@ invenio-cli assets watch
     changes. In addition, **be aware of the cache**, it might be a good idea
     to disable your cache or use an incognito window when developing web UI.
 
-This is a long running operation, so do not close the terminal. You
+This is a continually running operation, so do not close the terminal. You
```

Comment:
minor:

```suggestion
This is a continuous running operation, so do not close the terminal. You
```

Reply:
"continually" is the adverb applied to "running" (https://www.wordnik.com/words/continually).